rtt/os/gnulinux/traces/lttng_ust.h

```diff
@@ -0,0 +1,98 @@
+
+#undef TRACEPOINT_PROVIDER
+#define TRACEPOINT_PROVIDER orocos_rtt
+
+#if !defined(_LTTNG_UST_H) || defined(TRACEPOINT_HEADER_MULTI_READ)
+#define _LTTNG_UST_H
+
+#include <lttng/tracepoint.h>
+
+/* Allow to mapping a thread id to a name */
+TRACEPOINT_EVENT
+(
+    /* provider name */
+    orocos_rtt,
+    /* tracepoint name */
+    thread_init,
+    /* arguments passed to the tracepoint */
+    TP_ARGS(const char *, n),
+    /* the fields */
+    TP_FIELDS
+    (
+        ctf_string(name, n)
+    )
+)
+
+/* Allow to emulate a scope */
+TRACEPOINT_EVENT
+(
+    /* provider name */
+    orocos_rtt,
+    /* tracepoint name */
+    thread_scope,
+    /* arguments passed to the tracepoint */
+    TP_ARGS(unsigned, v),
+    /* the fields */
+    TP_FIELDS
+    (
+        ctf_integer(unsigned, value, v)
+    )
+)
+
+#ifndef __cplusplus
+#define TRACEPOINT_CONTEXT(provider, event, ...) \
+    TRACEPOINT_EVENT(provider, event, TP_ARGS(unsigned, v, const char *, s), TP_FIELDS(ctf_string(name, s) ctf_integer(unsigned, on, v)))
+#else
+#define TRACEPOINT_CONTEXT(provider, event, ...) \
```

Comment:
```cpp
In file included from /opt/orocos/kinetic/src/orocos_toolchain/rtt/rtt/os/traces.h:7:0,                                                                                                                                                
                 from /opt/orocos/kinetic/src/orocos_toolchain/rtt/rtt/Activity.cpp:46:                                                                                                                                                
/opt/orocos/kinetic/src/orocos_toolchain/rtt/rtt/os/gnulinux/traces/lttng_ust.h:82:57: warning: ISO C++11 requires at least one argument for the "..." in a variadic macro                                                             
 TRACEPOINT_CONTEXT(orocos_rtt, TaskContext_configureHook)                                                                                                                                                                             
                                                         ^
```
and many more.

Simply remove the `...`? I cannot find any usage of additional arguments.

Reply:
Confirmed that removing the unused full stops would work.